docs/integrations/redis.md

```diff
@@ -37,6 +44,9 @@ if __name__ == "__main__":
     asyncio.run(main())
 ```
 
+1. You can pass `capture_statement` to `logfire.instrument_redis()` to capture the Redis command. By default, it is set to `False`
```

Comment:
I don't like putting this info in these bubbles unless there's a good reason. Users are likely to not check the bubbles, and this is valuable info.